uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/tables/PagedTable.java

```diff
@@ -45,39 +55,43 @@
     @UiField
     public UberfireSimplePager pager;
 
+    @UiField(provided = true)
+    public Button pageSizesSelector;
+
+    private boolean showPageSizesSelector = false;
+    private PopupPanel popup = new PopupPanel(true);
```

Comment:
Please use BaseModal rather than PopupPanel.

We're trying to replace use of PopupPanel with BaseModal to improve (1) appearance (2) increase consistency.